src/lib/eas.ts

```diff
@@ -22,10 +22,13 @@ if (!EAS_SIGNER_PRIVATE_KEY) {
 }
 
 // Optimism address
-const eas = new EAS("0x4200000000000000000000000000000000000021")
+const eas =
+  process.env.NEXT_PUBLIC_ENV === "dev"
+    ? new EAS("0xC2679fBD37d54388Ce493F1DB75320D236e1815e")
+    : new EAS("0x4200000000000000000000000000000000000021")
 
 const provider = new ethers.AlchemyProvider(
-  "optimism",
+  process.env.NEXT_PUBLIC_ENV === "dev" ? "sepolia" : "optimism",
```

Comment:
added Sepolia schemas for testing and set `NEXT_PUBLIC_ENV = "dev"` for all preview deployments (feel free to configure as needed) @bswags @tinaszheng